IntroToCSharp/Pages/Activities/TheTypeSystem.razor

```diff
@@ -0,0 +1,151 @@
+@inherits CaptainCoder.Page
+@page "/activities/the-type-system"
+
+<MudText Typo="Typo.h4" Style="font-weight:bold">The Type System</MudText>
+
+<Section Title="Introduction">
+    <MudText>As we learned in the <MudLink Href="/activities/variables">Variables</MudLink> lesson, each variables must
```

Comment:
You should favor `<Keyword Word="type"/>` over `<Keyword Word="type"></Keyword>`

Reply:
done